critiquebrainz/frontend/templates/macros.html

```diff
@@ -51,3 +51,17 @@
     </script>
   {% endif %}
 {% endmacro %}
+
+{% macro review_credit(review) %}
+  {% if review.entity_type == 'event' %}
+    {% set credit_string = 'Event review by %(user)s' %}
+  {% elif review.entity_type == 'release_group' %}
+    {% set credit_string = 'Music review by %(user)s' %}
+  {% else %}
+    {% set credit_string = 'Review by %(user)s' %}
+  {% endif %}
+  {{ _(credit_string,
```

Comment:
@gentlecat Oh, right; I hadn't realized that, not having extracted the strings myself. Tested and resolved in 7876cd72814